Gems/Atom/RHI/Vulkan/Code/Source/RHI/Image.cpp

```diff
@@ -193,7 +193,11 @@ namespace AZ
                     // find the offset base on bound blocks
                     VkOffset3D offset;
                     offset.x = (boundBlockCount % blockCountPerRow) * imageGranularity.width;
-                    offset.y = (boundBlockCount / blockCountPerRow) * imageGranularity.height;
+                    // the offset.y need to consider 3D texture case. Consider the example if we have 64x64x64(width,height,layer) 3d
+                    // texture and the block size is 16x16x16. When boundBlockCount is 20, without (% blockCountPerColumn), we would get
+                    // offset.y to (20/4 *16) = 80. The later extent.height = mipSize.m_height-offset.y = ((unsigned)64-80) will be a huge
+                    // positive number, thus led to error in sparse Binding.
```

Comment:
I wouldn't put a bug description as a comment in code (but the description should be in the PR!).

Reply:
hey, thanks for the comments. I've updated the PR.